### PR TITLE
Fix mismatch in `ibz_kpoints` and add symmetry info in intergrate tests

### LINKED ISSUE #2472: symmetry bug: Refine the lattice parameters in STRU or use a different`symmetry_prec`
### Describe the bug

When I ran the following example using the latest development branch, I encountered the following error:
```
Optimized lattice type of reciprocal lattice cannot match the optimized real lattice. 
It is often because the inaccuracy of lattice parameters in STRU.
 
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
                         NOTICE                           
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
 Refine the lattice parameters in STRU or use a different`symmetry_prec`. 
 CHECK IN FILE : OUT.ABACUS/warning.log
 
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
                         NOTICE                           
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

  |CLASS_NAME---------|NAME---------------|TIME(Sec)-----|CALLS----|AVG------|PER%-------
                       total               0.16931        5         0.034     1e+02     %
 ----------------------------------------------------------------------------------------

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   RANK 0 PID 3356795 RUNNING AT iZ8vbdcc8xjb6er3bsa0u6Z
=   KILLED BY SIGNAL: 9 (Killed)
===================================================================================
```
[006_Ti15.zip](https://github.com/deepmodeling/abacus-develop/files/11503139/006_Ti15.zip)

### Expected behavior

_No response_

### To Reproduce

_No response_

### Environment

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by maki49**
fix https://github.com/deepmodeling/abacus-develop/issues/2472, and the symmetry-part error in https://github.com/deepmodeling/abacus-develop/issues/2474

The inaccuracy of STRU file sometimes makes the result of optimized-lattice-search mismatch between real and reciprocal space. That’s why I used reciprocal relation to force-match it in #2441 . But it destroyed the optimization and then cause the mismatch in other cases (like 002-4MoS2, 003-12Pt, 006-Ti15). 

So in this PR I try to search the optimized lattice again after using reciprocal relation. 
It can repair both of the previous-failed cases in my local test. 

For details, taking case 006-Ti15 as an example:
```
reciprocal-opt-latt from a direct search: (before #2441 ) (some cell-relax-generated cases failed)
0.170254 0.0982965 0
0 -0.196593 0
0 0 -0.107643
reciprocal-opt-latt from a direct use of reciprocal relation: (after #2441 ) (002-4MoS2, 003-12Pt, 006-Ti15 failed)
0.170254 -0.0982962 0
0 -0.196593 0
0 -0 -0.107643
reciprocal-opt-latt from searching optimized lattice again after using reciprocal relation: (now) (both are ok)
0.170254 0.0982965 0
-0.170254 0.0982962 -0
0 0 0.107643
```

### CODE REVIEW

**Comment by pxlxingliang** [comment]
@maki49 Please add this example to the integration test and modify the settings to make the time cost less than 30s or less

File: tests/integrate/tools/catch_properties.sh [hunk 1]
```diff
@@ -400,3 +401,11 @@ fi
 ttot=`grep $word $running_path | awk '{print $3}'`
 echo "totaltimeref $ttot" >>$1
 
+if ! test -z "$symmetry" && [ $symmetry == 1 ]; then
```
Comment (pxlxingliang) [inline tests/integrate/tools/catch_properties.sh:401]: you should add these codes before `echo "totaltimeref $ttot" >>$1`

**Review by pxlxingliang — CHANGES_REQUESTED** [review]

File: tests/integrate/tools/catch_properties.sh (hunk 1 above)
Comment (pxlxingliang) [reply tests/integrate/tools/catch_properties.sh:401]: The items after `totaltimeref` will not be checked

Comment (maki49) [reply tests/integrate/tools/catch_properties.sh:401]: OK, done.

**Comment by maki49** [comment]
The test failed because other symmetry-on cases' `result.ref` files have no symmetry infomation. 

I will add add symmetry info in all the `result.ref`s in those cases.

**Review by pxlxingliang — APPROVED** [review]

**Review by mohanchen — APPROVED** [review]

**Pull request merged**